src/SwedbankPay.Sdk.Infrastructure/Consumers/Consumer.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace SwedbankPay.Sdk.Consumers
 {
-    public class Consumer
+    public class Consumer : IConsumer
```

Comment:
Codacy found an issue: [This class can't be instantiated; make its constructor 'public'.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)